libraries/chain/controller.cpp

```diff
@@ -1331,7 +1331,7 @@ struct controller_impl {
          trace->except_ptr = std::current_exception();
          trace->elapsed = fc::time_point::now() - start;
 
-         if (auto dm_logger = get_deep_mind_logger()) {
+         if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
This is inside `push_scheduled_transaction`, which is not called for transient transactions.